recipes/novas/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "novas" %}
+{% set version = "3.1.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/42/95/a05bc35cb119925e10f9faa8a2bd17020b0a585744a38921a709acdd9a14/{{ name }}-{{ version }}.tar.gz
+  sha256: 6784780f03589996c2cd0e2b7e68afbec734d953010612ed0a45ace714761935
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    
+test:
+  imports:
+    - novas
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://pypi.org/project/novas/3.1.1.5/
+  summary: 'NOVAS is an integrated package of functions for computing various commonly needed quantities in positional astronomy.'
+  license: MIT
```

The text looks close to `NIST-PD` I would go with
```suggestion
  license: LicenseRef-USNO-PD
  license_family: PUBLIC-DOMAIN
```

---

Yes absolutely correct, done.